### DISCUSSION TITLE
Design improvement discussion - Aesthetic

### DISCUSSION THREAD

**Discussion by dzuk-mutant** [datafruits/datafruits#1343] [Ideas]
So, in various concepts I've shown, I've shown various tweaks to colour palettes, fonts and other visuals, and those tweaks were me basically trying to stay faithful to the vibes that I was picking up from the website, but they are just the vibes I was picking up. I've also tried to be somewhat conservative and avoiding too many changes because I didn't want to be stepping on someone else's vision.

A few of the ideas I've currently shown for reference:

![335874573-5ec78aab-c568-4bec-8ce0-f901933f8f27](https://github.com/datafruits/datafruits/assets/32108484/769625d0-5133-48e6-b6d6-4c3584a109ce)

![image](https://github.com/datafruits/datafruits/assets/32108484/0e99997a-10ff-44a9-919c-1767ab1c5f51)

I figured it would be a good idea to make a space to let others (the folks reading this) elaborate what kinds of things you'd like to see and are inspired by, so I can play around with aesthetic details (such as fonts, icons and certain approaches) for future iterations more confidently.

What you're inspired by for the site, what you'd like it to feel like in your most ideal scenario, etc. ^^

**Comment by SamRaymer** [comment]
I would love to see the green stay around as part of the UI, even if it's just for accents here and there.

**Reply by SamRaymer** [reply]
it just feels so _healthy_

**Comment by dudethatbe** [comment]
just to give some context on what's inspired us (mostly speaking for firedrill):
* https://www.lingscars.com/ - i really like how there's buttons everywhere and cheap animations
* https://battleofthebits.com/ - "shrimpo" is an adaptation from another site we really like. also love how old-internet it feels
* https://www.nts.live/ - like the navbar at the top and how they manage to jam so many things together without sub-menus. they also have good organization of archived shows, tracklists, etc.
* https://www.micromusic.net/ - inspiration for the datafruits messages aka microliner

@dzuk-mutant i keep repeating myself, but this always looks so good 😃 the new fonts really help. I think it's faithful to original idea, and even more so with the hand drawn look. 
The angled "datafruits.fm" logo with the pink background looks a little awkward on desktop, but i think it looks great on mobile.

**Reply by dzuk-mutant** [reply]
As mentioned on the Discord, these are great and thank you for sharing them ^^

I would like to explore other aesthetic approaches but the one I've done so far is the most natural to me as someone who has specialised in vectors and iconography a lot.

Yeah, I think the 'datafruits' text doesn't work on desktop as depicted either (hence the most recent concept in the nav discussion, though I do miss that nice hint of bright pink)
![image](https://github.com/datafruits/datafruits/assets/32108484/058f327b-b0df-429b-ac9f-207539c1720b)

I think other reasons I immediately jumped to hand-drawn vectors is:
- Their colour can be changed with themes without having to replace the assets (keeping the whole thing more maintanable while having more graphical elements)
- They are infinitely scalable so there doesn't need to be any concerns about display density

I do worry that it might flatten the appearance and make it feel too controlled. I think this will probably be an ongoing thing - maintaining a sense of lo-finess, silliness and scrappiness while trying to streamline the user experience to make it more accessible.

**Comment by dzuk-mutant** [comment]
![image](https://github.com/datafruits/datafruits/assets/32108484/e4a72b70-a3a8-4bbb-8373-57424b12f85d)

![image](https://github.com/datafruits/datafruits/assets/32108484/e50af7a0-764c-4ae6-adc8-96681084ecbf)


Here's some developments in aesthetic ideas (for the vector scribbles pathway anyway) from various recent sketches of the nav area:
- Hand-drawn (albeit slightly cleaned up to provide a balance of silliness and clarity) vector symbols and graphics.
- Hand-drawn, non-square SVG backgrounds that can stretch and be recoloured with the theme. These SVG backgrounds help provide aesthetic flair and also some extra semantic information.
    - While the background shapes are non-square, the contents still follow rigid keylines. This helps balance chaotic aesthetics and legibility.
- Soft, silly typefaces (currently using DynaPuff for headings and Comic Helvetic for body text)

**Comment by NicholasGannon** [comment]
Hi all,

I'm currently working on a repository that provides a curated list of useful **font and typography** resources for designers/developers! I'd really appreciate if you could check it out and give it a star!

https://github.com/NicholasGannon/Font-Resources

Thank you
